rxros2_cpp/README.md

```diff
@@ -45,19 +49,47 @@ This projects has received funding from the European Union's Horizon 2020 resear
 ## Dependencies
 The RxROS2 library depends on and uses the following software:<br>
 
-1. Ubuntu Bionic 18.04<br>
-2. ROS2 Eloquent Elusor<br>
-3. Reactive C++ v2<br>
-https://github.com/ReactiveX/RxCpp<br>
-Released under the Microsoft Open Source Code of Conduct.<br>
-The RxCpp library (header files) is installed as part of installing RxROS2<br>
+1. Ubuntu Focal Fossa 20.04 and ROS2 Foxy Fitzroy<br>
+2. Reactive C++ (RxCpp) v4.0<br>
```

Comment:
I would clarify although this is a dependency, it's not something users have to provide themselves. It's "taken care of" by the installation instructions provided below.